### PR TITLE
Feature/toast

### PR DESCRIPTION
**Pull request by 0x-shanks**
# Overview
* Add Toast Component
スクリーンショット 2022-05-18 0 58 50

# Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?

### CODE REVIEW

File: src/components/common/toast/Toast.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      </group>
+      <PerspectiveCamera makeDefault position={[10, 6, 10]} zoom={3} />
+    </>
+  );
+};
+
+export const Toast: FC<{}> = () => {
+  const { toast } = useCVoxelToast();
+  const [isMount, setIsMount] = useState<boolean>(false);
+
```
Comment (kitakaze-kan) [inline src/components/common/toast/Toast.tsx:56]: How about that?
```
if(!toast.isShow){
 return <></> ( or, return null)
}

Comment (kitakaze-kan) [reply src/components/common/toast/Toast.tsx:56]: I don’t try it yet so not sure that it works though.

**Comment by kitakaze-kan** [comment]
Thanks! almost LFGM!
Could you re-submit PR for `feature/v2_ui` ? 
(sorry it’s my mistake..)

File: src/components/common/toast/Toast.tsx (hunk 1 above)
Comment (0x-shanks) [reply src/components/common/toast/Toast.tsx:56]: I thought it looked good too, but when I actually moved it, the component disappeared the moment it was closed, so the animation was not rendered.

Comment (kitakaze-kan) [reply src/components/common/toast/Toast.tsx:56]: oh, I see. In that case that's okay. Thanks!

**Review by kitakaze-kan — APPROVED** [review]
approved

**Pull request merged**